DEVELOPMENT.md

```diff
@@ -166,7 +166,10 @@ ko apply -f config/
 # Optional steps
 
 # Configure outbound network for GKE.
-PROJECT_ID="my-gcp-project-id" ./hack/dev-patch-config-gke.sh my-k8s-cluster-name
+export PROJECT_ID="my-gcp-project-id"
+export K8S_CLUSTER_ZONE="my-cluster-zone" # Set K8S_CLUSTER_REGION for regional
+clusters
```

Comment:
Did you mean to wrap this?

Reply:
Nope. Chose to update the comments and add example for K8S_CLUSTER_REGION rather than remove the wrap so it renders well on small windows.